### ISSUE TITLE
Hotkeys don't work on Linux

### ISSUE THREAD

**Issue by lanthaler** [chabou/hyper-broadcast#4]
Hi,

Broadcasting itself works, but none of the hotkeys I tried (including the default ones) work. Additionally, the Super modifier seems to be dropped:

![image](https://user-images.githubusercontent.com/456407/47210130-6b14a080-d392-11e8-9a34-71616e50e954.png)

**Comment by chabou** [comment]
Thanks for your feedback.

If I understand your issue correctly, you can't specify `super` key for hotkeys?

If you use another keys, it works?

**Comment by lanthaler** [comment]
Sorry, somehow I missed your reply. If I specify `super` in a hotkey, it is silently dropped as shown in screenshot. Other modifier keys show up in the menu (tried, e.g., `shift+pagedown`) but they don't work either. The only way to activate broadcasting is to go to the menu and click on the desired entry.

**Comment by Herrox** [comment]
Hello @lanthaler @chabou
Hotkeys are also not working on `Windows` because of the `Command` modifier use. I just create a PR for fixing the problem on `Windows` and on `Linux`.

Otherwise, you could edit by yourself the file index.js and replace `Command` with `Control` or `CommandOrControl`.

Regards,

**Comment by lanthaler** [comment]
Thanks, that seems to be a separate issue though as I neither use `Command` nor `Control` as you can see in the screenshots in the issue description.

**Comment by lanthaler** [comment]
Yes, that didn't seem to work either on Linux if I remember correctly

**Comment by azwinabdul** [comment]
me too have some problem, i already tried with shift, alt but it didn't work. i use ubuntu 18.04.1 and hyper 2.1.2

**Comment by ac1dgoddess** [comment]
I'm also able to reproduce this issue on Linux Mint using Hyper 3.0.2 with hyper-broadcast. I can broadcast across panes if I manually go to File -> Broadcast Input - however, the hot keys & alternative combinations such as adding the `super` key do not work for me at all.

**Comment by misajimenezmx** [comment]
Any update for linux?

**Comment by chabou** [comment]
I'll try to tackle this but I don't use Linux right now.

A PR will be really welcome 🙏

**Comment by mathiasmag** [comment]
I cannot create a PR because these are technologoes I do not have experience with.

However, getting access to Linux cannot be the biggest obsticle. I understand lack of time to maintain though.

Being able to do this from the keyboard would make the tool at least twice as useful for me. Probably more like 10x.

**Comment by asdf23** [comment]
I suspect that you would need to unbind any existing keystrokes that might be conflicting.  At the operating system level gnome or kde I'd suggest that you verify that these keys are not assigned...
